libs/core/src/lib/avatar-group/directives/avatar-group-internal-overflow-button.directive.ts

```diff
@@ -0,0 +1,50 @@
+import { Directive, EmbeddedViewRef, inject, Input, OnDestroy, TemplateRef, ViewContainerRef } from '@angular/core';
+import { AvatarGroupItemRendererDirective } from './avatar-group-item-renderer.directive';
+
+@Directive({
+    selector: '[fdAvatarGroupInternalOverflowButton]',
+    standalone: true
+})
+export class AvatarGroupInternalOverflowButtonDirective implements OnDestroy {
+    /** @hidden */
+    @Input('fdAvatarGroupInternalOverflowButton')
+    set hiddenItems(value: AvatarGroupItemRendererDirective[] | null) {
+        const hiddenItems = value || [];
+        if (hiddenItems.length > 0) {
+            this.show(hiddenItems);
+        } else {
+            this.hide();
+        }
+    }
+
+    /** @hidden */
+    readonly templateRef: TemplateRef<{ $implicit: AvatarGroupItemRendererDirective[] }> = inject(TemplateRef);
```

Comment:
You can also provide `static ngTemplateContextGuard` for type support in templates